consoles/sshVirtsh.pm

```diff
@@ -352,9 +353,16 @@ sub _copy_image_vmware ($self, $name, $backingfile, $file_basename, $vmware_open
 }
 
 sub _copy_image_else ($self, $file, $file_basename, $basedir) {
-    $self->run_cmd(sprintf("rsync -av '$file' '$basedir/%s'", $file_basename)) && die 'rsync failed';
+    if (-e $file_basename && defined which 'rsync') {    # utilize asset possibly cached by openQA worker
+        my %c = $self->get_ssh_credentials;
+        bmwqemu::diag "Syncing '$file' directly from worker host to $c{hostname}";
+        CORE::GLOBAL::system "RSYNC_PASSWORD='$c{password}' rsync -av '$file_basename' '$c{username}\@$c{hostname}:$basedir/$file_basename'";
```

Comment:
There should be some check whether the rsync was successful.

Reply:
It will die because we use autodie here. So the error handling is equivalent to the ` && die 'rsync failed'` in the existing code branch.